packages/browser/__tests__/jose/IsomorphicJoseUtility.test.ts

```diff
@@ -23,41 +23,32 @@ import { describe, it } from "@jest/globals";
 import {
   decodeJWT,
   generateCodeVerifier,
-  generateJWK,
   signJWT,
 } from "../../src/jose/IsomorphicJoseUtility";
-
-describe("generateJWK", () => {
-  it("can generate a RSA-based JWK", async () => {
-    const key = await generateJWK("RSA");
-    expect(key.kty).toEqual("RSA");
-  });
-
-  it("can generate an elliptic curve-based JWK", async () => {
-    const key = await generateJWK("EC", "P-256");
-    expect(key.kty).toEqual("EC");
-  });
-});
+import {
+  generateDpopKey,
+  generateRsaKey,
+} from "@inrupt/oidc-dpop-client-browser";
 
 describe("signJWT/decodeJWT", () => {
```

Comment:
The unit tests in this `describe()` block look like they belong down in the DPop library (or just removed from here if there are similar/same tests down in that library already). (But then, is the RSA key generation really DPop (or DCR)-specific (i.e. does it belong in `oidc-client-dpop-browser` at all...)?

Reply:
Signing and decoding the JWT will be moved to the `oidc-client-dpop-browser` module in a future PR, the current one only moves key generation around. The RSA key generation isn't DPoP/DCR-specific, but moving it in the oidc-specific module isolates the crypto code from the rest of the library.